activetable.css

```diff
@@ -51,8 +56,37 @@ tr.odd td.right-answer {
 }
 input[type="text"] {
     font-size: 1em;
+    line-height: 1.5em;
     border: 0;
     padding: 0 10px;
     margin: 0;
-    background-color: rgba(0, 0, 0, 0);
+    box-sizing: border-box;
+    display: block;
+    width: 100%;
+    background-color: transparent;
+}
+input[type="text"]:hover {
+    outline: 2px solid #a0a0ff;
```

Comment:
Would you like it darker or brighter?  The colour should have [quite good contrast compared to white](http://www.color-hex.com/color/a0a0ff), but of course it depends on your screen.